src/components/Kv/KvPillToggle.vue

```diff
@@ -26,45 +26,71 @@ export default {
 	},
 	methods: {
 		pillToggled(key) {
-			this.$emit('pillToggled', key);
+			this.selected = key;
+			this.$emit('pill-toggled', key);
+		}
+	},
+	created() {
+		if (this.options.length > 0) {
+			const option = this.options[0];
+			this.selected = option.key;
 		}
 	}
 };
 
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 @import "settings";
 
 $form-border-radius: rem-calc(3);
 
-.pill-option {
-	display: inline-block;
-	border: 1px solid $subtle-gray;
-	cursor: pointer;
-}
+.pill-toggle {
+	font-size: rem-calc(14);
+	height: rem-calc(34);
+	width: 100%;
 
-.pill-option:first-child {
-	border-top-left-radius: $form-border-radius;
-	border-bottom-left-radius: $form-border-radius;
-}
+	span {
+		background: #F7F7F7;
+		border: rem-calc(1) solid #C4C1C6;
+		color: #808080;
```

Comment:
What are the vars for these colors?

Reply:
@mcstover Unfortunately, there are no vars for the L54-56 vars.

Reply:
Please create in _settings.scss

Reply:
@mcstover Color vars added to `_settings.scss` in a new block.

Reply:
Thank you!